shell/src/main/java/tachyon/shell/command/LoadCommand.java

```diff
@@ -65,25 +63,22 @@ void runCommand(TachyonURI path) throws IOException {
    */
   private void load(TachyonURI filePath) throws IOException {
     try {
-      TachyonFile fd = mTfs.open(filePath);
-      FileInfo fInfo = mTfs.getInfo(fd);
-      if (fInfo.isFolder) {
-        List<FileInfo> files = mTfs.listStatus(fd);
-        Collections.sort(files);
```

Comment:
The previous sort on `FileInfo` wasn't a logical sort, so I don't think its necessary.